sources/packages/web/src/views/student/StudentAccountActivity.vue

```diff
@@ -10,6 +10,9 @@
         :routeLocation="{ name: StudentRoutesConst.STUDENT_DASHBOARD }"
       />
     </template>
+    <tab-container :enableCardView="false" class="mb-6">
+      <student-account-restrictions
+    /></tab-container>
```

Comment:
The component doesn't need to be wrapped by `tab-container`. Even it is not required in the existing code for `student-scholastic-standing-limited-history`.